GitHub-Test-DHprojects/emilyGitHub3.md

```diff
@@ -0,0 +1,5 @@
+# Emily Canvas Discussion
+## Student DH Projects 
+This project took a look at Hamilton's timeline of events, soundtrack, character interactions, and historical accuracy. One of my favorite visuals provided was the line graph of repeated phrases. It was so interesting because it took only a small amount of the most frequent words and was able to see how many times it occurred in each song. I like how in some songs it was used for a different reason than others. Some of the phrases that were used the most somehow related to a bigger overall theme of the musical. 
+
+This project gave me the idea to analyze a script or dialogue and relate what words are used, back to the piece's overall purpose for being created, and how accurately it was displayed through its chosen words/phrases.
```

Comment:
Ooooh, I did something similar (last semester?) for another class Dr. B taught. I compared two shows made by the same creative director to see how similar they were!